src/circuits/runcircuit.jl

```diff
@@ -163,9 +163,11 @@ Apply a quantum gate to an MPS/MPO.
 function applygate!(M::Union{MPS,MPO},
                     gatename::String,
                     sites::Union{Int,Tuple};
+                    cutoff=1E-15,
+                    maxdim=1_000,
```

Comment:
1000 is a fairly small bond dimension, do we need to put a limit here? If anything, I would want to make it a warning if a bond dimension exceeds some memory, for example.

Reply:
Also, this reminds me that it would be good to add a feature to write MPS tensors that are not being used to disk (like you can do with ITensor's DMRG function), which helps with memory limitations.